.circleci/config.yml

```diff
@@ -57,6 +57,15 @@ jobs:
       - run:
           name: Publish @next
           command: npm publish --tag next --access public
+      - run:
+          name: Login to Docker Hub
+          command: echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
+      - run: 
+          name: Build :next
+          command: docker build --pull --rm -f "Dockerfile" -t m365pnp/cli-microsoft365:next "." --build-arg CLI_VERSION=next
```

Comment:
Since we're not using a specific version of CLI here, but rather the `@next` tag, we need to verify that CI will indeed use the version that we have _just_ published rather than the one used as `@next` previously.

Reply:
Agreed, updated to pull the version from the package after it is updated by the script which is then passed as a build arg.